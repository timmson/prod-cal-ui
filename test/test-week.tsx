import React from "react"
import renderer from "react-test-renderer"
import Week from "../src/week"

describe("Week should", () => {

	test("be equal to snapshot", () => {
		const w = {
			number: 1,
			days: [
				{
					type: "inactive",
					date: 1
				}
			]
		}

		const component = renderer.create(<Week week={w}/>)

		expect(component.toJSON()).toMatchSnapshot()
		component.unmount()
	})

})