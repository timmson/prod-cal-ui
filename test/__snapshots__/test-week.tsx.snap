// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Week should be equal to snapshot 1`] = `
<tr>
  <td
    className="weekNum"
  >
    1
  </td>
  <td
    className="inactive"
  >
    1
  </td>
</tr>
`;
